README.rst

```diff
@@ -25,240 +25,51 @@
 
 .. after_badges_rst_tag
 
-===========================
-Introduction to libEnsemble
-===========================
+=====================================================================
+libEnsemble: A complete toolkit for dynamic ensembles of calculations
+=====================================================================
 
-libEnsemble is a Python_ toolkit for coordinating workflows of asynchronous and dynamic ensembles of calculations.
+Adaptive, portable, and scalable software to connect "deciders" to experiments or simulations.
 
-libEnsemble helps users take advantage of massively parallel resources to solve design,
-decision, and inference problems and expands the class of problems that can benefit from
-increased parallelism.
+• **Adaptive ensembles**: Generate parallel tasks on-the-fly based on previous computations.
```

Comment:
Heres my effort:

- **Dynamic ensembles**: Generate new tasks on-the-fly based on previous computations.
- **Extreme scaling**: Run identical scripts on or across_ laptops, clusters, and leadership-class machines.
- **Dynamic resource management**: Dynamically assign and reassign CPU/GPU resources to tasks.
- **Application monitoring**: Ensemble members can run, monitor, and cancel apps.
- **Heterogeneous resources** Automatic detection and correct assignment on Nvidia, AMD and Intel GPUs.
- **Low start-up cost**: No additional background services or processes required.

Thoughts?

Reply:
In Dynamic resources

we have dropped: "partitions of any size" (e.g. nodes) - which to my mind made clear it included nodes, but I'm guessing thats obvious already

Reply:
Coordinated data-flow between tasks - does this mean via the manager? I'm not sure about this one.

Reply:
> Here's my effort:

Looks good! But I wonder if there's a way of combining the Dynamic resource management and Heterogeneous resources sections?

> In Dynamic resources

Yeah, that's what I was thinking. Plus "nodes" is covered in my mind by "extreme scaling".

> Coordinated data-flow between tasks - does this mean via the manager? I'm not sure about this one.

I'm trying to emphasize that the tasks don't exist independently, each in a "bubble" (which is the case for most multi-task workflow systems). They can communicate state with the manager and each other

Reply:
Oh yeah, I'm also trying to avoid redundant adjectives whenever possible.